internal/api/features/_topics.json

```diff
@@ -14,7 +14,6 @@
     "taint",
     "nodelabel",
     "byres",
-    "tmpol",
-    "schedattrwatch"
+    "tmpol"
```

Comment:
typo. Please add `[release-4.15]` to the title of the commit message to convey the fact this change exists only in this branch

Reply:
not typo, sorry, misread. But please update the commit title anyway